src/mumble/AudioOutput.cpp

```diff
@@ -588,29 +588,56 @@ bool AudioOutput::mix(void *outbuff, unsigned int frameCount) {
 						aop->pfVolume[s] = -1.0;
 				}
 
+				if (!aop->piOffset) {
+					aop->piOffset = std::make_unique<unsigned int []>(nchan);
+					for (unsigned int s = 0; s < nchan; ++s) {
+						aop->piOffset[s] = 0;
+					}
+				}
+
 				for (unsigned int s = 0; s < nchan; ++s) {
-					const float dot = bSpeakerPositional[s]
-										  ? connectionVec.x * speaker[s * 3 + 0] + connectionVec.y * speaker[s * 3 + 1]
-												+ connectionVec.z * speaker[s * 3 + 2]
-										  : 1.0f;
-					const float str   = svol[s] * calcGain(dot, len) * volumeAdjustment;
+					const float dot   = bSpeakerPositional[s]
+											? connectionVec.x * speaker[s * 3 + 0] + connectionVec.y * speaker[s * 3 + 1]
+                                                + connectionVec.z * speaker[s * 3 + 2]
+											: 1.0f;
+					// Volume on the ear opposite to the sound should never reach 0 in the real world.
+					// The gain is multiplied by 19/20 and 0.05 is added. This will have the effect
+					// of bringing the lowest value up to 0.05, while keeping the highest value at 1.
```

Comment:
```suggestion
					// The gain is multiplied by 19/20 and 1/20 is added. This will have the effect
					// of bringing the lowest value up to 1/20, while keeping the highest value at 1.
```